docs/zkapps/tutorials/05-common-types-and-functions.mdx

```diff
@@ -294,123 +337,126 @@ If a program is too large to fit into these constraints, it can be broken up int
 
 You can use [Merkle trees](../snarkyjs-reference/classes/MerkleTree) to manage large amounts of data within a circuit. The power of Merkle trees is demonstrated in the [05-common-types-and-functions/src](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/05-common-types-and-functions/src) reference project for this tutorial. See the [BasicMerkleTreeContract.ts](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/05-common-types-and-functions/src/BasicMerkleTreeContract.ts) contract and [main.ts](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/05-common-types-and-functions/src/main.ts) that demonstrates how contracts interact with Merkle trees and how to construct them.
 
```

Comment:
well.. no big deal to delete, but you could leave it in there if you used an `ignore` directive like for skipped commands. Do we have that?
EDIT: nvm I saw that you used it elsewhere

Reply:
Yes we do have `ignore`. I can also add it back, I don't mind. I just felt like it was odd that we only mentioned the import for `MerkleTree`, but didn't talk about importing all the other previous primitives. So rather than adding an `import` to all the previous sections I removed this one :sweat_smile:

Reply:
ok!